package.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
```

Comment:
Let's add `set -eou pipefail` here. It's good practice for shell scripts when you can do it (see: http://redsymbol.net/articles/unofficial-bash-strict-mode/) and in this particular case it will cause the script to error on lines 8-10 if the caller does not set DISTRO, APP, or VERSION en vars as required by the script.